src/components/Signup.js

```diff
@@ -75,6 +75,7 @@ class Signup extends React.Component {
                 <input
                   className="formkit-input"
                   name="email_address"
+                  aria-label="Your email address"
```

Comment:
Do you mind reverting changes to other files please?

Reply:
Hi @gaearon, I reverted changes in those two files by using `git checkout 66fbf1e~1 -- [file]`. I probably should have used `git revert` though. Can you confirm if further changes need to be made on my end?